tools/harness-thci/OpenThread_WpanCtl.py

```diff
@@ -2347,8 +2347,8 @@ def MGMT_ACTIVE_SET(
 
             if xMasterKey is not None:
```

Comment:
Should we fix `xMasterKey` too?

Reply:
Is `MGMT_ACTIVE_SET()` called by test harness? Will updating `xMasterKey` affect compatibility with test harness? If not, then we should update this as well.